LogMonitor/src/LogMonitor/EventMonitor.cpp

```diff
@@ -488,7 +496,7 @@ EventMonitor::PrintEvent(
                 status = GetLastError();
 
                 logWriter.TraceError(
-                    Utility::FormatString(L"Failed to render event. Error: %lu", status).c_str()
+                    util.FormatString(L"Failed to render event. Error: %lu", status).c_str()

```

Comment:
why the change form `Utility` from the _static_ approach we've been using?

Reply:
This change was to enable reference a non-static method (FormatEventLineLog) but I have made method static and reverted the changes done one the previous approach.